app/src/components/Tabs/Activity/direct-measure-table.tsx

```diff
@@ -130,13 +130,23 @@ const DirectMeasureTable: FC<DirectMeasureTableProps> = ({
                   </Text>
                 </Td>
                 <Td isNumeric isTruncated>
-                  {convertKgToTonnes(activity?.activityData?.co2_amount)}
+                  {/*Direct measure entries are collected in tonnes by default*/}
```

Comment:
I think our default is to save all gas amounts in kg actually? At least that was the intent...